demo/addons/sofiaconsole/Console.cs

```diff
@@ -88,6 +134,45 @@ public override void _Input(InputEvent @event)
             _commandInput.ReleaseFocus();
             _commandSendButton.ReleaseFocus();
         }
+
+        UpdateUi();
+    }
+
+    private void UpdateUi(bool close = false)
+    {
+        if (!Open) return;
+
+        foreach (var node in _suggestionsContent.GetChildren())
+        {
+            node.QueueFree();
+        }
+        
+        if (close) return;
+        if (_suggestions.Count == 0)
+        {
+            _suggestionsPanel.Hide();
+            return;
+        }
+
+        _suggestionsPanel.Show();
+        foreach (var text in _suggestions)
+        {
+            var newLabel = new Label
+            {
+                Text = text,
+                Theme = new Theme
+                {
+                    DefaultFontSize = 30
+                }
+            };
+
+            if (_currentSuggestionIndex >= 0 && _suggestions.Count > 0 && _currentSuggestionIndex < _suggestions.Count && _suggestions[_currentSuggestionIndex] == text)
+            {
+                newLabel.AddThemeStyleboxOverride("normal", GD.Load<StyleBoxFlat>("res://addons/sofiaconsole/StyleBox.tres"));
```

Comment:
We should probably give this asset a proper name, StyleBox does not indicate what it is used for.